### DISCUSSION TITLE
Design pattern for new sub pages

### DISCUSSION THREAD

**Discussion by timokonkwo** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#4] [Outreachy 2022] [answered]
@alswajiab 
@BatoolMM 

Good day. My name is Tim and I'm an outreachy intern who will be contributing to this great project.

I have proceeded to create the sub pages required of us but my question is this:

Must I maintain the same header, footer and other color and design from the original website (https://osc-ksa.com/) or can I come up with a different design on my own?

**Comment by BatoolMM** [comment, marked as answer]
Hi @timokonkwo, thank you for your interest in our community!
Yes, you will have to maintain the same header, footer and other colors and design from the original website.

Let me know if you have any other questions!
